ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -275,20 +275,40 @@ echo "$(date) - all clusteroperators are done progressing."
 i=0
```

Comment:
please move the clusteroperator wait *after* the nodes ready check please

Reply:
done